sdk/contracts.ts

```diff
@@ -133,6 +138,12 @@ const CanonicalAddresses: Record<
   [KnownContracts.CONNEXT]: {
     "1.0.0": "0x7dE07b9De0bf0FABf31A188DE1527034b2aF36dB",
   },
+  [KnownContracts.PACKER]: {
```

Comment:
Maybe it would make sense to prefix with `ROLES_` to communicate that these contracts belong together

Reply:
(we should've done the same for v1 permissions, but whatever, we can start doing it better now...)

Reply:
Yes it makes sense. I had of course noticed this. But since I did not introduce that TypeScript enum (and would've never introduced it without namespace), and since changing would be a breaking change, I just did not do it

Reply:
Make its technically not even a breaking change, because since we only keep bytecode for latest, Permissions should be a remove, instead of a rename. So then this could be ROLES_PACKER, ROLES_INTEGRITY